Vegeting/Screens/RegisterUserProfile/UserProfileViewController.swift

```diff
@@ -175,31 +182,39 @@ final class UserProfileViewController: UIViewController {
     @objc
     private func textDidChangeForLabel() {
         guard let text = nicknameTextField.text else { return }
+        var textLength = text.count
         
         //maxLength 초과 시 키보드를 아래로 내려주는 역할
         if text.count >= nicknameMaxLength {
             nicknameTextField.resignFirstResponder()
         }
         
+        //maxLength 이상의 글자를 붙여넣을 경우 잘라주는 역할
         if text.count > nicknameMaxLength {
             let index = text.index(text.startIndex, offsetBy: nicknameMaxLength)
```

Comment:
```suggestion
        //maxLength 이상의 글자를 붙여넣을 경우 잘라주는 역할
        if textLength > nicknameMaxLength {
            let index = text.index(text.startIndex, offsetBy: nicknameMaxLength)
```
기존에 선언된 변수를 활용하면 더 좋을 거 같습니다~

Reply:
[수정](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/133/files#diff-d95417c458a8528974d98adab570fc3e6ffedf546ea53c010fbdf052c46d30d8R188-R190)하였습니다~